velox/row/CompactRow.h

```diff
@@ -36,6 +37,15 @@ class CompactRow {
   /// 'buffer' must have sufficient capacity and set to all zeros.
   int32_t serialize(vector_size_t index, char* buffer);
 
+  /// Serializes rows at specified index range into 'buffer' at given offset.
+  /// 'buffer' must have sufficient capacity and set to all zeros.
+  /// The size of 'offsets' and indexRange must be the same.
+  /// The value of 'offsets' will be updated by the actual bytes written.
+  void serialize(
+      const IndexRange& indexRange,
```

Comment:
IndexRange is just a pair of start and end row numbers. Perhaps, it would be more readable to use 2 separate parameters. Otherwise, readers need to lookup IndexRange.